### ISSUE TITLE
CI is failing

### ISSUE THREAD

**Issue by mattheworiordan** [ably/ably-java#110]
See https://travis-ci.org/ably/ably-java/jobs/112552681

We should focus on getting that stable again as a priority.

**Comment by paddybyers** [comment]
Fixed: https://ci.ably.io/job/ably-java/193/

@gokhanbarisaker: I was getting this in the log when our CI runs:

````
Running test: test auth_clientid_null_mismatch(io.ably.lib.test.rest.RestAuthTest)

io.ably.lib.test.rest.RestSuite > io.ably.lib.test.rest.RestAuthTest.auth_clientid_null_mismatch FAILED
    java.lang.NoSuchMethodError
````

I had also got this error when running in development, because of a version mismatch between the `org.hamcrest` classes in the hamcrest package, mockito, and Junit itself. The fix was to order the classpath so that the hamcrest package is referenced first.

How are you running things so you weren't getting this issue?
